### DISCUSSION TITLE
Rainbow Oaks Renewed

### DISCUSSION THREAD

**Discussion by BrandonPotter0** [DynamicTreesTeam/DynamicTrees#983] [Compatibility Addon Suggestions]
https://modrinth.com/mod/rainbow-oaks-3

Minecraft 1.20.1/Forge compatibility please.

There appears to be only the 1 Tree that seems to spawn naturally in Oak Forests, and other biomes where Oak trees spawn.

![image](https://github.com/user-attachments/assets/4f30d05c-6c3c-4345-961c-49d3690bc444)
![image](https://github.com/user-attachments/assets/434821a8-eef3-47c4-b0d2-a3938ae81e32)

**Comment by Groupix05** [comment]
Done
https://www.curseforge.com/minecraft/mc-mods/dynamic-trees-rainbow-oak-renewed

**Reply by Groupix05** [reply]
It spawns only in flower forest biomes
